packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -128,23 +149,36 @@ function NetworkListbox({
       </Listbox.Button>
 
       <Listbox.Options className="absolute z-20 mt-2 rounded-xl bg-white shadow-[0px_4px_12px_#9e9e9e]">
-        {options.map((option, index) => (
-          <Listbox.Option
-            key={option.chainID}
-            value={option}
-            className={twMerge(
-              'flex h-12 cursor-pointer items-center space-x-2 px-4 hover:bg-blue-arbitrum hover:bg-[rgba(0,0,0,0.2)]',
-              getOptionClassName(index)
-            )}
-          >
-            <img
-              src={getOptionImageSrc(option)}
-              alt={`${getNetworkName(option.chainID)} logo`}
-              className="w-8"
-            />
-            <span>{getNetworkName(option.chainID)}</span>
-          </Listbox.Option>
-        ))}
+        {options.map((option, index) => {
+          const isDisabled = isOptionDisabled(option.chainID)
+
+          return (
+            <Tooltip
+              key={option.chainID}
+              show={isDisabled}
+              content={getDisabledNetworkPairTooltip(option)}
+              wrapperClassName="w-full"
+              theme="dark"
+            >
+              <Listbox.Option
+                value={option}
+                className={twMerge(
+                  'flex h-12 cursor-pointer items-center space-x-2 px-4 hover:bg-blue-arbitrum hover:bg-[rgba(0,0,0,0.2)]',
+                  getOptionClassName(index),
+                  isDisabled ? 'pointer-events-none opacity-40' : ''
+                )}
+                disabled={isDisabled}
+              >
+                <img
+                  src={getOptionImageSrc(option)}
```

Comment:
The Ethereum logo looks squashed in the dropdown:

<img width="279" alt="Screen Shot 2022-10-26 at 14 21 13" src="https://user-images.githubusercontent.com/20543771/198024682-e26754e2-ea2e-479b-aee7-08fad2fdba2b.png">